Sources/Models/Views/Label.swift

```diff
@@ -88,7 +91,7 @@ public struct Label: IBDecodable, ViewProtocol {
             subviews:                                  container.childrenIfPresent(of: .subviews),
             text:                                      container.attributeIfPresent(of: .text),
             textAlignment:                             container.attributeIfPresent(of: .textAlignment),
-            textColor:                                 container.elementIfPresent(of: .textColor),
+            textColor:                                 colorsContainer?.withAttributeElement(.key, CodingKeys.textColor.stringValue),
```

Comment:
I would suggest below to extract `textColor`.
```swift
let colors: [Color] = try container.elements(of: .color)
colors.first { $0.key == "textColor" }
colors.first { $0.key == "backgroundColor" }
```

Reply:
I used the same technique as in https://github.com/IBDecodable/IBDecodable/blob/5e07f7b761aed10e8035c34bdae30c44cf5a9755/Sources/Models/Documents/Placeholder.swift.
This was suggested in the related issue #52 .

Reply:
I see. That makes sense.